spec/tbd_tests_spec.rb

```diff
@@ -722,38 +722,31 @@
     # edge types, the automated selection will be deterministic. With 2 or more
-    # edge types sharing the exact same PSI factor (e.g. 0.3 W/K per m), the
-    # final selection of edge type becomes less obvious. It is not randomly
+    # edge types sharing the exact same PSI-factor (e.g. 0.3 W/K per m), the
+    # final edge type selection becomes less obvious. It is not randomly
     # selected, but rather based on the (somewhat arbitrary) design choice of
     # which edge type is processed first in psi.rb (line ~1300 onwards). For
-    # instance, fenestration perimeter joints are treated before corners or
-    # parapets. When dealing with equal hash values, Ruby's Hash "key" method
+    # instance, fenestration perimeters are treated before corners or parapets.
+    # When dealing with equal hash values, Ruby's Hash "key" method
     # returns the first key (i.e. edge type) that matches the criterion:
     #
     #   https://docs.ruby-lang.org/en/2.0.0/Hash.html#method-i-key
     #
     # From an energy simulation results perspective, the consequences of this
-    # pseudo-random choice are insignificant (i.e. same PSI-value). For UA'
+    # pseudo-random choice are insignificant (i.e. ~same PSI-factor). For UA'
     # comparisons, the situation becomes less obvious in outlier cases. When a
-    # reference value needs to be generated for the edge described above, TBD
-    # retains the original autoselected edge type, yet applies reference PSI
-    # values (e.g. code). So far so good. However, when "(non thermal bridging)"
-    # is retained as a default PSI design set (not as a reference set), all edge
-    # types will necessarily have 0 W/K per metre as PSI-values. Same with the
-    # "efficient (BETBG)" PSI set (all but one type at 0.2 W/K per m). Not
-    # obvious (for users) which edge type will be selected by TBD for multi-type
-    # edges. This also has the undesirable effect of generating variations in
-    # reference UA' tallies, depending on the chosen design PSI set (as the
-    # reference PSI set may have radically different PSI-values depending on
-    # the pseudo-random edge type selection). Fortunately, this effect is
-    # limited to the somewhat academic PSI sets like "(non thermal bridging)" or
-    # "efficient (BETBG)".
+    # reference value needs to be generated for a given edge, TBD retains the
+    # original autoselected edge type, yet applies reference PSI values (e.g.
+    # "code"). So far so good. However, when "(non thermal bridging)" is
+    # retained as a default PSI design set (not as a reference set), all edge
+    # types will necessarily have PSI-factors of 0 W/K per metre. To minimize
+    # the issue, slight variations (e.g. +/- 0.000001 W/K per inear meter) have
+    # been added to TBD built-in PSI-factor sets (where required). Without this
+    # fix, undesirable variations in reference UA' tallies may occur.
```

Comment:
Rehashing overview of TBD edge type determination (when based solely on equal PSI-factors).